README.md

```diff
@@ -29,7 +29,7 @@ Once dune is available the executable can be locally changed and run with the
 following:
 
 ```
-$ dune exec cooltt                      # from the `cooltt` top-level directory
+$ opam exec cooltt
```

Comment:
I think this may call the pinned version, but that's almost never what you want when developing. So, let's not make this change.